### PR TITLE
TET-94 Right toolbar

### PR DESCRIPTION
**Pull request by GurgenAvagyan**
- Create class `RightToolbar`
- Add class `ToggleSwitch` for displaying toggle switch

image

### CODE REVIEW

**Review by VadimST04 — APPROVED** [review]

File: src/main/java/org/cmps/tetrahedron/config/WindowProperties.java [hunk 1]
```diff
@@ -8,22 +12,65 @@
...
+    private static int logicalHeight;
+
+    public static final int MIN_WIDTH = 1000;
+    public static final int MIN_HEIGHT = 800;
 
-    public static int getWidth() {
-        return width;
+    private static boolean changed = true;
+
+    static {
```
Comment (HappyMary16) [inline src/main/java/org/cmps/tetrahedron/config/WindowProperties.java:23]: this may be done without static block

Comment (HappyMary16) [reply src/main/java/org/cmps/tetrahedron/config/WindowProperties.java:23]: if you add comments, do it in English

File: src/main/java/org/cmps/tetrahedron/utils/Utils.java [hunk 2]
```diff
@@ -0,0 +1,16 @@
+package org.cmps.tetrahedron.utils;
+
+import javafx.scene.text.Font;
+
+public class Utils {
```
Comment (HappyMary16) [inline src/main/java/org/cmps/tetrahedron/utils/Utils.java:5]: And what is this utils class doing?
Let's give it more descriptive name

**Review by HappyMary16 — CHANGES_REQUESTED** [review]

**Review by HappyMary16 — APPROVED** [review]

**Review by ut5tw — APPROVED** [review]
please resolve conflicts

**Pull request merged**